src/condor_contrib/bosco/bosco_cluster

```diff
@@ -839,10 +841,13 @@ fi
 start_ssh
 
 # Quickly test the ssh
-qmgr_out=`ssh -o "PasswordAuthentication=no" $remote_host "pwd" 2>/dev/null`
+ssh_opts='-o "PasswordAuthentication=no"'
+[ $debug -eq 0 ] || ssh_opts="$ssh_opts -vvvvv"
+qmgr_out=$(ssh $ssh_opts $remote_host "pwd" 2>&1)
```

Comment:
```suggestion
qmgr_out=$(ssh $ssh_opts "$remote_host" "pwd" 2>&1)
```

Not super important here since it's probably expected that the user will give a remote host that does not contain whitespace or shell glob characters, but since it is uninspected user input the safest thing is generally to quote it.  I get that all over this script `$remote_host` is used unquoted so maybe this "improvement" won't match the rest of the script.

I mean, i suppose no user in their right mind would run this script with
`--add "my.remote.host /bin/rm -rf /"`
... right?  ;)

Nasty hobbitses.